.eslintrc.json

```diff
@@ -4,6 +4,16 @@
         "node": true
     },
     "extends": "eslint:recommended",
+    "ecmaFeatures": {
+      "sourceType": "module"
+    },
+    "parserOptions": {
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "jsx": true,
+        "experimentalObjectRestSpread": true
+      }
+    },
```

Comment:
This is the wrong place to have these options. The root of the project is the `node` specific `.eslintrc.json` file while the files in `/client`, `/tests`, and `/tests/client` have their specific overrides. This applies the module syntax across the entire node codebase.